test/functional/services/dashboard/add_panel.ts

```diff
@@ -79,32 +81,42 @@ export class DashboardAddPanelService extends FtrService {
 
   async toggleFilterPopover() {
     this.log.debug('DashboardAddPanel.toggleFilter');
-    await this.testSubjects.click('savedObjectFinderFilterButton');
+    const filtersHolder = await this.find.byClassName('euiSearchBar__filtersHolder');
```

Comment:
How about wrapping saved object finder click functionality into a page service? Looks like the same logic is used in 3 places and its non-trivial.